frame/identity/src/lib.rs

```diff
@@ -451,9 +456,112 @@ decl_error! {
 		InvalidTarget,
 		/// Too many additional fields.
 		TooManyFields,
+		/// Maximum amount of registrars reached. Cannot add any more.
+		TooManyRegistrars,
 }
 }
 
+/// Functions for calcuating the weight of dispatchables.
+mod weight_for {
+	use frame_support::weights::{RuntimeDbWeight, Weight};
+
+	/// Weight calculation for `set_identity`.
+	pub(crate) fn set_identity(
+		db: RuntimeDbWeight,
+		judgements: impl Into<Weight>,
+		extra_fields: impl Into<Weight>
+	) -> Weight {
+		db.reads_writes(1, 1)
+			+ db.reads_writes(1, 1) // balance ops
```

Comment:
This should be excluded because balance read/write targets the `System.Account` storage item, which will already be updated this block because a user submitted an extrinsic, and their nonce/fee will be charged.

Reply:
Same for the other balance operations in this PR. It would only apply if the user whose balance is changing is NOT the caller.